### ISSUE TITLE
[Add request] Browser locker

### ISSUE THREAD

**Issue by iam-py-test** [blocklistproject/Lists#428]
URLs you wish to be added:
```
0.0.0.0 helooworld.us
0.0.0.0 www.helooworld.us
# Which redirects to 
0.0.0.0 rockcabs247.in
```
Why you believe this should be added:
This website locks the user’s browser with a fake alert claiming to be from Windows Defender, and tricks the user into calling a phone number controlled by the attacker (who probably will then do a tech support scam).
See https://github.com/DandelionSprout/adfilt/issues/228 for more information 
Add to list:
Fraud, scams, malware, and phishing? (It is all of those)
Other info you think we should know:
The homepage of the first domain redirects to Facebook but a certain url (in the issue report above) redirects to the second. 
This was verified yesterday using a VM.

**Comment by blocklistproject** [comment]
I just tested all these and they do not redirect to a fake alert. It may have changed or been taken down since you tested. I will add the redirects.

**Comment by iam-py-test** [comment]
> I just tested all these and they do not redirect to a fake alert. It may have changed or been taken down since you tested. I will add the redirects.

@blocklistproject checking...
Are you in the US?

**Comment by blocklistproject** [comment]
@iam-py-test yes

**Comment by iam-py-test** [comment]
@blocklistproject I think you need a certain url

**Comment by iam-py-test** [comment]
@blocklistproject `rockcabs247.in` is the real browser exploit (it does seem to have been taken down to avoid blocklisting; maybe the person behind it noticed it being blocked by major AVs and blocklists & went into hidding)

**Comment by blocklistproject** [comment]
For now I have added the redirects and will close this but will check on the other every now and again.

**Comment by iam-py-test** [comment]
I will try to keep an eye on it & will open another issue if it comes back
